### ISSUE TITLE
(Forge) Fluidlogged blocks carry over fluids when moved using Create

### ISSUE THREAD

**Issue by sentwayfarer** [Leximon/Fluidlogged#8]
https://user-images.githubusercontent.com/95097230/222554966-dab3d501-91e6-4331-af96-8279abfd3972.mp4

I saw the other issue and I'm kinda disappointed it's not maintained anymore, but if fixing it is a possibility, please do so

**Comment by ExpensiveKoala** [comment]
Create would have to handle this, or a mixin hacking this functionality would have to be implemented.
Create determines whether to put a water block or air block here: https://github.com/Creators-of-Create/Create/blob/b50b22938b1a173d8b6e21dad63b265dbd2cbe87/src/main/java/com/simibubi/create/content/contraptions/components/structureMovement/Contraption.java#L956
And determines whether to waterlog a block being placed in water here: https://github.com/Creators-of-Create/Create/blob/b50b22938b1a173d8b6e21dad63b265dbd2cbe87/src/main/java/com/simibubi/create/content/contraptions/components/structureMovement/Contraption.java#L1037

**Comment by sentwayfarer** [comment]
Yep, noticed that a while ago. I attempted a pull request on Create but I couldn't figure out how since I'm not that good at Java.

**Comment by Leximon** [comment]
I think it's better to let Create handle this issue. I've tried fixing it using mixins but the solution is inconvenient in my opinion. Especially for version 1.19.3 since Create doesn't exist for this version yet.
I'll let this issue open for now and probably create a PR on Create's side to address this issue but first, I have to rerelease the Forge version of this mod.

**Comment by sentwayfarer** [comment]
> I think it's better to let Create handle this issue. I've tried fixing it using mixins but the solution is inconvenient in my opinion. Especially for version 1.19.3 since Create doesn't exist for this version yet. I'll let this issue open for now and probably create a PR on Create's side to address this issue but first, I have to rerelease the Forge version of this mod.

Alright! If possible, can you do it on 1.18.2 and have them or you port it over to 1.19.2? I'm working with 1.18.2.

**Comment by sentwayfarer** [comment]
New issue related to this one, I'm using the renewed Forge port, and now when moving contraptions into fluids it just voids the fluid instead of fluidlogging said block.

**Comment by Leximon** [comment]
Sort of fixed in 2.0.0-beta1 but it's not really optimal, so I'll let this issue open
